### ISSUE TITLE
PATCH the EVC to change queue_id is not triggering the redeploy

### ISSUE THREAD

**Issue by italovalcy** [kytos-ng/mef_eline#346]
Hi,

Even though queue_id is one of the attributes on the `attributes_requiring_redeploy` variable, when PATCHing the queue_id on some EVCs, that will not trigger redeploy... not sure what happened:

```
# for evc in 756523c310f343 65b2bfd371f249 6877fee190a64d 06d735cdac454a; do curl -s -X GET http://127.0.0.1:8181/api/kytos/mef_eline/v2/evc/$evc | jq -r '.queue_id'; done
0
0
0
0
# for evc in 756523c310f343 65b2bfd371f249 6877fee190a64d 06d735cdac454a; do curl -X PATCH http://127.0.0.1:8181/api/kytos/mef_eline/v2/evc/$evc -d '{"queue_id":1}' -H 'Content-type: application/json'; done
...
# for evc in 756523c310f343 65b2bfd371f249 6877fee190a64d 06d735cdac454a; do curl -s -X GET http://127.0.0.1:8181/api/kytos/mef_eline/v2/evc/$evc | jq -r '.queue_id'; done
1
1
1
1
# grep '2023-07-03 14:24:' /var/log/kytos.log | grep -v socket.io
2023-07-03 14:24:07,237 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121070) 127.0.0.1 - - [03/Jul/2023 14:24:07] "PATCH /api/kytos/mef_eline/v2/evc/756523c310f343 HTTP/1.1" 200 -
2023-07-03 14:24:07,285 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121071) 127.0.0.1 - - [03/Jul/2023 14:24:07] "PATCH /api/kytos/mef_eline/v2/evc/65b2bfd371f249 HTTP/1.1" 200 -
2023-07-03 14:24:07,334 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121072) 127.0.0.1 - - [03/Jul/2023 14:24:07] "PATCH /api/kytos/mef_eline/v2/evc/6877fee190a64d HTTP/1.1" 200 -
2023-07-03 14:24:07,378 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121073) 127.0.0.1 - - [03/Jul/2023 14:24:07] "PATCH /api/kytos/mef_eline/v2/evc/06d735cdac454a HTTP/1.1" 200 -
2023-07-03 14:24:13,228 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121080) 127.0.0.1 - - [03/Jul/2023 14:24:13] "GET /api/kytos/mef_eline/v2/evc/756523c310f343 HTTP/1.1" 200 -
2023-07-03 14:24:13,296 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121081) 127.0.0.1 - - [03/Jul/2023 14:24:13] "GET /api/kytos/mef_eline/v2/evc/65b2bfd371f249 HTTP/1.1" 200 -
2023-07-03 14:24:13,362 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121082) 127.0.0.1 - - [03/Jul/2023 14:24:13] "GET /api/kytos/mef_eline/v2/evc/6877fee190a64d HTTP/1.1" 200 -
2023-07-03 14:24:13,417 - INFO [werkzeug] [_internal.py:225:_log] (Thread-121083) 127.0.0.1 - - [03/Jul/2023 14:24:13] "GET /api/kytos/mef_eline/v2/evc/06d735cdac454a HTTP/1.1" 200 -
```

**Comment by italovalcy** [comment]
Workaround available: send a new round of requests to force redeploy:
```
# for evc in 756523c310f343 65b2bfd371f249 6877fee190a64d 06d735cdac454a; do curl -X PATCH http://127.0.0.1:8181/api/kytos/mef_eline/v2/evc/$evc/redeploy; done
{"response":"Circuit 756523c310f343 redeploy received."}
{"response":"Circuit 65b2bfd371f249 redeploy received."}
{"response":"Circuit 6877fee190a64d redeploy received."}
{"response":"Circuit 06d735cdac454a redeploy received."}

# grep '2023-07-03 14:26:' /var/log/kytos.log | grep deployed
2023-07-03 14:26:07,752 - INFO [kytos.napps.kytos/mef_eline] [evc.py:734:deploy_to_path] (Thread-121194) EVC(756523c310f343, Vlan_344_MIA-Loop_SCL) was deployed.
2023-07-03 14:26:09,461 - INFO [kytos.napps.kytos/mef_eline] [evc.py:793:setup_failover_path] (thread_pool_app_115) Failover path for EVC(756523c310f343, Vlan_344_MIA-Loop_SCL) was deployed.
2023-07-03 14:26:09,529 - INFO [kytos.napps.kytos/mef_eline] [evc.py:734:deploy_to_path] (Thread-121204) EVC(65b2bfd371f249, Vlan_345_345_MIA-Loop_SCL) was deployed.
2023-07-03 14:26:11,185 - INFO [kytos.napps.kytos/mef_eline] [evc.py:793:setup_failover_path] (thread_pool_app_23) Failover path for EVC(65b2bfd371f249, Vlan_345_345_MIA-Loop_SCL) was deployed.
2023-07-03 14:26:11,239 - INFO [kytos.napps.kytos/mef_eline] [evc.py:734:deploy_to_path] (Thread-121218) EVC(6877fee190a64d, Vlan_346_MIA-Loop_Sp3) was deployed.
2023-07-03 14:26:12,066 - INFO [kytos.napps.kytos/mef_eline] [evc.py:793:setup_failover_path] (thread_pool_app_1) Failover path for EVC(6877fee190a64d, Vlan_346_MIA-Loop_Sp3) was deployed.
2023-07-03 14:26:13,107 - INFO [kytos.napps.kytos/mef_eline] [evc.py:734:deploy_to_path] (Thread-121237) EVC(06d735cdac454a, Vlan_347_346_MIA-Loop_Sp3) was deployed.
2023-07-03 14:26:13,839 - INFO [kytos.napps.kytos/mef_eline] [evc.py:793:setup_failover_path] (thread_pool_app_82) Failover path for EVC(06d735cdac454a, Vlan_347_346_MIA-Loop_Sp3) was deployed.
```

**Comment by viniarck** [comment]
@italovalcy thanks for reporting this. I haven't tried to reproduce on `2022.3` yet, but on `2023.1` (currently unreleased) it's redeploying as expected:

```
2023-07-03 12:09:55,028 - INFO [kytos.napps.kytos/flow_manager] [main.py:624:_send_flow_mods_from_request] (AnyIO worker thread) Send FlowMod from request dpid: 00:00:00:00:00:00:00:03,
 command: add, force: False, flows_dict: {'flows': [{'match': {'in_port': 1}, 'cookie': 12279057277719076169, 'actions': [{'action_type': 'push_vlan', 'tag_type': 's'}, {'action_type': 
'set_vlan', 'vlan_id': 3}, {'action_type': 'output', 'port': 3}, {'action_type': 'set_queue', 'queue_id': 2}], 'owner': 'mef_eline', 'table_group': 'epl', 'table_id': 0, 'priority': 100
00}, {'match': {'in_port': 3, 'dl_vlan': 3}, 'cookie': 12279057277719076169, 'actions': [{'action_type': 'pop_vlan'}, {'action_type': 'output', 'port': 1}, {'action_type': 'set_queue', 
'queue_id': 2}], 'owner': 'mef_eline', 'table_group': 'evpl', 'table_id': 0, 'priority': 20000}], 'force': False}
2023-07-03 12:09:55,036 - INFO [uvicorn.access] [httptools_impl.py:506:send] (MainThread) 127.0.0.1:38830 - "POST /api/kytos/flow_manager/v2/flows/00%3A00%3A00%3A00%3A00%3A00%3A00%3A03 
HTTP/1.1" 202
2023-07-03 12:09:55,042 - INFO [kytos.napps.kytos/mef_eline] [evc.py:751:deploy_to_path] (AnyIO worker thread) EVC(67f9896f1edd49, epl) was deployed.
2023-07-03 12:09:55,042 - INFO [uvicorn.access] [httptools_impl.py:506:send] (MainThread) 127.0.0.1:38780 - "PATCH /api/kytos/mef_eline/v2/evc/67f9896f1edd49 HTTP/1.1" 200
2023-07-03 12:09:55,048 - INFO [uvicorn.access] [httptools_impl.py:506:send] (MainThread) 127.0.0.1:38842 - "POST /api/kytos/pathfinder/v3/ HTTP/1.1" 200
```

Although looking into the code on `2022.3.1` as you said, it was supposed to redeploy too, I wonder if it ended up with bad data in the data model due to validation of previous requests for some reason, if you still have this scenario can you double check if any other error showed up or you see inconsistent any invalid value on `queue_id`. 

Also, since you have this workaround of reploy do you think we can temporarily leave with this since on `2023.1` it's expected to be working, including with better validations that will avoid bad data in the first place? Let me know.

**Comment by viniarck** [comment]
I'll go ahead and close this since it's been fixed since `2023.1` and we decided won't fix on `2022.3`

Thanks, guys.
